tsl/test/sql/transparent_decompression_ordered_index.sql.in

```diff
@@ -132,3 +132,102 @@ SET enable_seqscan = FALSE;
 SET enable_seqscan = TRUE;
 
 \ir include/transparent_decompression_constraintaware.sql
+
+-- github bug 2917 with UNION ALL that references compressed ht
+CREATE TABLE entity
+(
+  oid bigint PRIMARY KEY,
+  type text,
+  name text
+);
+
+INSERT INTO entity values(10, 'VMEM', 'cpu');
+
+CREATE TABLE entity_m2 
+( 
+timec         timestamp with time zone  , 
```

Comment:
can you make this NOT NULL to get rid of the NOTICE in the test output

Reply:
removed.